internal/infrastructure/providers/openweather_provider.go

```diff
@@ -0,0 +1,125 @@
+package providers
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"net/url"
+	"weather-forecast/internal/domain/models"
+	"weather-forecast/internal/infrastructure/apperrors"
+	"weather-forecast/internal/infrastructure/logger"
+)
+
+var metricUnits = "metric"
+
+const NotFoundOpenWeather = "404"
```

Comment:
```suggestion
const NotFoundOpenWeatherErrorCode = "404"
```
І можна зробити приватною константою всередині GetWeatherByCity

Reply:
Виправив

Reply:
Я потім ці константи в тестах перевикористовую, тому все ж залишу їх глобальними. Тести сьогодні ввечері постараюсь переробити